chart/README.md

```diff
@@ -49,5 +49,5 @@ instructions.
 [3]: https://github.com/cloudfoundry-incubator/kubecf/blob/master/doc/Contribute.md#customization
 [4]: https://quarks.suse.dev/docs/quarks-operator/concepts/variables/#explicit-variables
 [5]: https://quarks.suse.dev/docs/quarks-operator/concepts/variables/#implicit-variables
-[6]: https://kubecf.suse.dev/docs/getting-started/kubernetes-deploy/
-[7]: https://kubecf.suse.dev/docs/
+[6]: https://kubecf.io/docs/getting-started/kubernetes-deploy/
```

Comment:
This link is broken; perhaps we want https://kubecf.io/docs/deployment/ here? (Pretty sure it was broken before the domain switch, too.)

Reply:
Looks like it moved to https://kubecf.io/docs/deployment/kubernetes-deploy/.

Reply:
Fixed.